lfucg-exactions/client/js/components/PlatForm.js

```diff
@@ -524,6 +558,15 @@ function mapDispatchToProps(dispatch, params) {
                                 dispatch(formUpdate(update2));
```

Comment:
Is there a good reason to name this "update2"?  Could this be something more descriptive?